pkg/util/workerqueue/workerqueue.go

```diff
@@ -37,6 +37,32 @@ const (
 	workFx = time.Second
 )
 
+// debugError is a marker type for errors that that should only be logged at a Debug level.
+// Useful if you want a Handler to be retried, but not logged at an Error level.
+type debugError struct {
+	err error
+}
+
+// NewDebugError returns a debugError wrapper around an error.
+func NewDebugError(err error) error {
+	return &debugError{err: err}
+}
+
+// Error returns the error string
+func (l *debugError) Error() string {
+	if l.err == nil {
+		return "nil"
```

Comment:
Should this be return "<nil>"?

https://play.golang.org/p/gbFHmpOtdtJ

Reply:
I went back and forth on that. In the rare occurrence in which it could be `nil` I thought it might be handy to be able to easily see that. Empty string so often disappears amongst other things.

Maybe it should be "<nil>" like the playground then?

Reply:
I meant to put "<nil>", I guess I didn't paste the middle part in though.

Reply:
When I go to edit your comment, I see you wrote `<nil>`, which gets escaped by Github! Cool. I'll make this change :smile:

Reply:
.....and done!